src/Pay/Pay.php

```
@@ -218,7 +218,7 @@ public function createPaymentMethod(string $customerId, string $type, array $det
      * @param  array<mixed>  $address
      * @return array<mixed>
      */
-    public function updatePaymentMethodBillingDetails(string $paymentMethodId, string $type, string $name = null, string $email = null, string $phone = null, array $address = null): array
+    public function updatePaymentMethodBillingDetails(string $paymentMethodId, string $type, ?string $name = null, ?string $email = null, ?string $phone = null, ?array $address = null): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$type'. (undefined)

(UnusedFormalParameter)